tck/bridge-tck-main-portlet/pom.xml

```diff
@@ -85,6 +85,7 @@
 		<portlet.container>liferay</portlet.container>
 		<integration.container>${portlet.container}</integration.container>
 		<portal.version.major>${liferay.version.major}</portal.version.major>
+		<tck.test.case.class>LiferayTckTestCase</tck.test.case.class>
```

Comment:
@stiemannkj1 This assumes Liferay is the default, and uses the "pluto" profile to override. I think we normally default to Pluto, and then override with Liferay specific stuff in the "liferay" profile. No?

Reply:
@ngriffin7a, we default to testing in Liferay to be consistent with the integration tests.